FDW - Release branch (#3331)
EnterpriseDB/docs: product_docs/docs/hadoop_data_adapter/2/hadoop_rel_notes/hadoop_rel_notes_2.3.0.mdx

---
title: "Version 2.3.0"
---

Enhancements, bug fixes, and other changes in Hadoop Foreign Data Wrapper 2.3 include:

| Type        | Description                                                                                                                                                                                                                             |
| ----------- |---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | Feature     | Push down `ORDER BY` to remote HDFS servers: If possible, push the `ORDER BY` clause to the remote server. This approach provides the ordered result set from the foreign server, which can help to enable an efficient merge join.     |

Reviewer: ```suggestion
| Feature     | When possible, we push the `ORDER BY` clause to the remote server. This approach provides the ordered result set from the foreign server, which can help to enable an efficient merge join.     |
```

> | Feature     | Push down `LIMIT`/`OFFSET` to remote HDFS servers: Where possible, perform `LIMIT` and `OFFSET` operations on the remote server. This reduces network traffic between local Postgres and remote HDFS/Hive servers.                      |

Reviewer: ```suggestion
| Feature     | When possible, we perform `LIMIT` and `OFFSET` operations on the remote server. This reduces network traffic between local Postgres and remote HDFS/Hive servers.                      |
```
